test/system-tests/repl.test.ts

```diff
@@ -47,7 +47,7 @@ describe('repl', () => {
 					assert.include(output, expect, `output ${output} does not contain ${expect}`);
 				});
 				test(':n', () => {
-					const expect = normalized ? normalizedAstToMermaid(normalized.ast) : '';
+					const expect = normalized ? normalizedAstToMermaid(normalized.ast, {}) : '';
```

Comment:
Only comment: please make the config itself optional, it feels weird to be required to pass in empty braces if you do not want to change the configuration + it breaks the api